src/chrome/extension/manifest.json

```diff
@@ -38,14 +38,11 @@
     "notifications",
     "https://www.uproxy.org/oauth-redirect-uri*",
     "webRequest",
-    "https://uproxy.org/request/*",
-    "https://uproxy.org/offer/*",
     "https://a0.awsstatic.com/",
     "webRequestBlocking"
   ],
   "web_accessible_resources": [
     "icons/*",
```

Comment:
Some more testing required? Chrome won't load the extension with this trailing comma.